### DISCUSSION TITLE
Translating the game on Android

### DISCUSSION THREAD

**Discussion by tiil357** [doukutsu-rs/doukutsu-rs#323] [Ideas]
Open a space to be able to translate the game on Android into various languages, made by the community. As in the PC version; That would be great. Thank you (for the effort).

**Comment by KLONOA9X6** [comment]
This function has been implemented.
![image](https://github.com/doukutsu-rs/doukutsu-rs/assets/95112579/3f02ccd4-6a3c-4c2a-95d7-d1e30847abc3)
![image](https://github.com/doukutsu-rs/doukutsu-rs/assets/95112579/1954539b-9f6c-4572-81d4-ef1cc5bc511b)
![image](https://github.com/doukutsu-rs/doukutsu-rs/assets/95112579/957517f9-9c9a-4017-980e-a4ceccb10ee6)
![image](https://github.com/doukutsu-rs/doukutsu-rs/assets/95112579/928da0c9-41b6-434f-9490-460f7914ce28)

**Reply by edwarly999plus** [reply]
How i can do that?
